en/patterns/user-profile.md

```diff
@@ -11,19 +11,18 @@ Use the User Profile Pattern to display user-related information, either as a fu
 <img class="responsive-img" src="../images/profile_demo.png" srcset="../images/profile_demo@2x.png 2x" />
 <img class="responsive-img" src="../images/profile_small.png" srcset="../images/profile_small@2x.png 2x" />
 
-An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar, Text, and Inputs that constitute its layout.
+An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar and Inputs that constitute its layout.
```

Comment:
Let's add an image of these editable variants below this paragraph

Reply:
Done. Please have a look